divi/src/BlockFactory.cpp

```diff
@@ -5,15 +5,16 @@
 #include <BlockMemoryPoolTransactionCollector.h>
 #include <CoinstakeCreator.h>
 #include <timedata.h>
+#include "main.h"
 
```

Comment:
Was there nothing smaller to include in place of main.h ? I doubt we need the entirety of main. We can tackle it incrementally though 👍

Reply:
replaced by:
#include <pow.h>
#include <chain.h>